Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3-alpine as base
+
+WORKDIR /usr/src/app
+
+RUN wget -q -O- https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+ENV PATH=/root/.poetry/bin:${PATH}
+
+FROM base as production
+COPY ./ ./
+RUN poetry install --no-dev -n
```

Comment:
To take advantage of Docker's layer caching, you could copy just the toml files, run poetry install and then copy the rest of the files.

This would mean that rebuilding the image after changing some app code will not re-run `poetry install`.